cinema/tests/test_cinema_hall_api.py

```diff
@@ -31,9 +31,7 @@ def test_get_cinema_halls(self):
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(response.data[0]["name"], blue_hall["name"])
         self.assertEqual(response.data[0]["rows"], blue_hall["rows"])
-        self.assertEqual(
-            response.data[0]["seats_in_row"], blue_hall["seats_in_row"]
-        )
+        self.assertEqual(response.data[0]["seats_in_row"], blue_hall["seats_in_row"])
```

Comment:
In `test_get_cinema_halls`, the test assumes the order of cinema halls returned by the API. Consider sorting the response data or using a more flexible assertion to avoid test failures due to ordering issues.